src/main/java/banking/database/DBConnector.java

```diff
@@ -0,0 +1,38 @@
+package banking.database;
+
+
+import org.sqlite.SQLiteDataSource;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DBConnector {
+
+    private String dbName;
+    private String url;
+    private SQLiteDataSource dataSource = new SQLiteDataSource();
+    private final String CREATE_TABLE = """
```

Comment:
That's not a constant. The constant format is next
```
private **static** final String CREATE_TABLE = "";
```
Also, the constants should be declared first in the class.

Reply:
fixed